jaxrs-api/src/main/java/javax/ws/rs/ext/Extension.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2018 Markus KARG. All rights reserved.
```

Comment:
I guess this should be 2019?

Reply:
I can change it to 2019, but indeed the code was written in 2018 already.